scrapy/pqueues.py

```diff
@@ -167,10 +132,10 @@ class DownloaderAwarePriorityQueue(SlotBasedPriorityQueue):
     _DOWNLOADER_AWARE_PQ_ID = 'DOWNLOADER_AWARE_PQ_ID'
 
     @classmethod
-    def from_crawler(cls, crawler, qfactory, startprios={}):
+    def from_crawler(cls, crawler, qfactory, startprios=None):
```

Comment:
Sorry, I don't understand why you replace `{}` by `None` here. `{}` was used in accord with https://github.com/scrapy/queuelib/blob/574ef504d217993dd5c098068ad43852868b5293/queuelib/pqueue.py#L24

Reply:
it is just a style change - mutable default arguments are considered bad style and a source of possible errors; in pqueues it was different, as tuple is immutable. None works the same here, if I'm not mistaken.